### DISCUSSION TITLE
How to reset the chunk state by using the nvme-cli cmd

### DISCUSSION THREAD

**Discussion by Theodore0414** [MoatLab/FEMU#139] [Q&A]
I use ocssd2.0. After entering the following command `sudo nvme write /dev/nvme0n1 -s 0 -c 1 -z 8192`, how should I reset it? When I enter `sudo nvme write /dev/nvme0n1 -s 0 -c 1 -z 8192` again, it shows **[FEMU] Err: Error IO processed! submit-io: Interrupted system call.**

**Comment by huaicheng** [comment]
Better debug it on FEMU side and see how it handles this case. It might be FEMU's improper logic to deal with resets, though I haven't spent time digging yet.
